targets/targets.json

```diff
@@ -584,7 +584,9 @@
         "features": ["LWIP", "STORAGE"],
         "release_versions": ["2", "5"],
         "device_name": "MK64FN1M0xxx12",
-        "bootloader_supported": true
+        "bootloader_supported": true,
+        "vector_table_size": 1024,
+        "flash_erase_size": 4096
```

Comment:
I think it's possible, but at the moment I don't know how to do it. Patches to arm-pack-manager welcome!